packages/cli/tests/test.ts

```diff
@@ -0,0 +1,99 @@
+import { cliArgv, generatedFiles, generatorName, GeneratorResults } from '@ngxs/cli/tests/config';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+import * as nodePlop from 'node-plop';
+import { expect } from 'chai';
```

Comment:
If possible, please could you use jasmine as opposed to chai and mocha for these tests?

Reply:
because Karma (Jasmine) does not work with node environment, we usage ts-mocha for testing CLI

https://github.com/angular/angular-cli/issues/8357

node-plop usage child_process

Reply:
https://github.com/amwmedia/node-plop/issues/81